.github/workflows/build-push-images.yml

```diff
@@ -85,6 +71,25 @@ jobs:
         with:
           mask-password: true
           registries: 311462405659
+
+      - name: Trivy Image Vulnerability Scanner for ${{ matrix.ecr_repository }}
+        id: trivy_scan
+        uses: aquasecurity/trivy-action@0.24.0
+        env:
+          TRIVY_DB_REPOSITORY: ${{ steps.login_ecr.outputs.registry }}/trivy-db-public-ecr/aquasecurity/trivy-db:2
```

Comment:
there's also a java db worth pulling through the cache

https://github.com/ministryofjustice/opg-modernising-lpa/blob/9b9fa10b9f6186641548a9c25a347c7ce640f0b9/.github/workflows/docker_job.yml#L104

Reply:
Cheers, I've added that now 👍🏽